allennlp/modules/conditional_random_field.py

```diff
@@ -255,26 +255,13 @@ def _joint_likelihood(self,
         else:
             score = 0.0
 
-        # Broadcast the transition scores to one per batch element
-        broadcast_transitions = self.transitions.view(1, num_tags, num_tags).expand(batch_size, num_tags, num_tags)
```

Comment:
You've removed the need for `num_tags`, so you need to remove the variable definition above on line 245 (`batch_size, sequence_length, _ = logits.size()`).